js/source/geojson_worker_source.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+var GeojsonWorkerSource = require('./geojson_worker_base');
+module.exports = function (self) {
+    self.registerWorkerSource('geojson', Object.create(GeojsonWorkerSource));
+};
```

Comment:
@lucaswoj Yeah, I think this can/will get cleaned up as part of addressing https://github.com/mapbox/mapbox-gl-js/pull/2667#discussion_r65789437